Index: src/lemonmenu.cpp
===================================================================
--- src/lemonmenu.cpp	(revision 50)
+++ src/lemonmenu.cpp	(working copy)
@@ -28,6 +28,7 @@
 #include <algorithm>
 #include <SDL/SDL_rotozoom.h>
 #include <SDL/SDL_thread.h>
+#include <windows.h>
 
 #define UPDATE_SNAP_EVENT 1
 
@@ -347,8 +348,15 @@
    cmd.replace(pos, 2, rom);
    ll::log << debug << "handle_run: " << cmd << endl;
 
+   // get handle of forground window
+   HWND hw = GetForegroundWindow();
+   
    int exit_code = system(cmd.c_str());
-
+   
+   // open iconified window and set it as foreground
+   OpenIcon(hw);
+   SetForegroundWindow(hw);
+   
    // only increment the games play counter if emulator returned success
    if (exit_code == 0) {
       
